src/store/modules/plex/helpers/PlexClient.js

```diff
@@ -344,7 +349,7 @@ module.exports = function PlexClient() {
     const serverId = serverObject.clientIdentifier;
     const address = serverObject.chosenConnection.address;
     const port = serverObject.chosenConnection.port;
-    const protocol = serverObject.chosenConnection.protocol;
+    const protocol = serverObject.chosenConnection.protocol; //Check where this comes from
```

Comment:
Use `//TODO: {comment}` for things that you want to follow-up on. Will make it easy to find them later to remove plus most IDEs support listing these either natively or by an extension.

Reply:
i actually figured that out and forgot to remove